art/attacks/evasion/carlini.py

```diff
@@ -530,10 +531,10 @@ def __init__(
         targeted: bool = False,
         learning_rate: float = 0.01,
         max_iter: int = 10,
-        max_halving: int = 5,
-        max_doubling: int = 5,
-        eps: float = 0.3,
-        batch_size: int = 128,
```

Comment:
Is removal of `batch_size` intended?

Reply:
Yes, this is intended because the new version follows closely the author's original implementation and does not work with mini-batches.